src/datasources/results/ResultsDataSourceBase.ts

```diff
@@ -93,6 +79,31 @@ export abstract class ResultsDataSourceBase extends DataSourceBase<ResultsQuery>
     }
   }
 
+  async queryProducts(
```

Comment:
I think we need to get in the habit of adding the `public` modifier for our public members. Has there been a tech debt item to update the linter with that?